Library/Homebrew/extend/ENV/shared.rb

```diff
@@ -196,7 +197,7 @@ def ncurses_define
 
   # @private
   def userpaths!
-    paths = self["PATH"].split(File::PATH_SEPARATOR)
+    paths = PATH.new(self["PATH"]).to_a
```

Comment:
@alyssais, I have to take a look at this method again, since in this case it is used twice.

But yes, `Array += PATH` should be an `Array` due to `to_ary`.